asar-teensy-firmware/TestScripts/Combined/MotorandProximity/MotorandProximity.ino

```diff
@@ -0,0 +1,69 @@
+/* TestScript1.ino
+ *  Author: Aaron Mallabar
+ *  Date:   January 18,2017
+ *  Description: Start Testing some of the capabilities of our MSD prototype Robot PCB.
+ *  1. Motor Movement
+ *  
+ */
+
+ #include "MOTOR.h"
+ #include "Proximity_Sensor.h"
+ #include "PITimer.h"
+
+const int LED = 13;
+const int Period = .1;
+bool Flag = 0;
+
+void callback();
+
+ ASAR::MOTOR myMOTOR;
+ ASAR::Proximity_Sensor myPing;
+
+ void callback();
```

Comment:
Remove one of these function declarations for `callback`.

Reply:
Removed